_common.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -euxfo pipefail;
+
+USERNAME="${DOCKER_USERNAME:-jackfirth}"
+
+find_images () {
+    docker images --format '{{.Repository}}:{{.Tag}}' | grep "^${USERNAME}/racket:[[:digit:]]" | sort
```

Comment:
How does the `[[:digit:]]` syntax work? Is that a special grep thing?

Reply:
Yep, it's a [pre-defined character class](https://www.gnu.org/software/grep/manual/html_node/Character-Classes-and-Bracket-Expressions.html) in grep representing individual digits, so that expression matches all the racket images that are followed by at least one digit. I was imagining a situation where someone contributing to the library could have, for example, a `racket:experimental` image that they wouldn't want to upload so the purpose of the digit is to filter those kinds of things out.